src/apis/notice-boards/services/notice-boards.service.ts

```diff
@@ -0,0 +1,103 @@
+import { Injectable } from '@nestjs/common';
+import { CreateNoticeBoardDto } from '../dto/create-notice-board.dto';
+import { DataSource, Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { NoticeBoard } from '@src/entities/NoticeBoard';
+import { NoticeBoardDto } from '../dto/notice-board.dto';
+import { HttpInternalServerErrorException } from '@src/http-exceptions/exceptions/http-internal-server-error.exception';
+import { COMMON_ERROR_CODE } from '@src/constants/error/common/common-error-code.constant';
+import { QueryHelper } from '@src/helpers/query.helper';
+import { FindNoticeBoardListQueryDto } from '../dto/find-notice-board-list-query.dto';
+import { NoticeBoardsItemDto } from '../dto/notice-boards-item.dto';
+import { NoticeBoardHistoryService } from '../notice-board-history/services/notice-board-history.service';
+
+@Injectable()
+export class NoticeBoardsService {
+  private readonly LIKE_SEARCH_FIELD: readonly (keyof Pick<
+    NoticeBoardDto,
+    'title'
+  >)[] = ['title'];
+
+  constructor(
+    private readonly queryHelper: QueryHelper,
+    private readonly noticeBoardHistoryService: NoticeBoardHistoryService,
+    private readonly dataSource: DataSource,
+    @InjectRepository(NoticeBoard)
+    private readonly noticeBoardRepository: Repository<NoticeBoard>,
+  ) {}
+  async create(userId: number, createNoticeBoardDto: CreateNoticeBoardDto) {
```

Comment:
```suggestion
  ) {}
  
  async create(userId: number, createNoticeBoardDto: CreateNoticeBoardDto) {
```
성격이 다른 구문은 개행이 있으면 가독성에 좋습니다.